src/components/book/book.service.js

```diff
@@ -0,0 +1,197 @@
+import bookMdl from "../../models/book.js";
+import helper from "../../utils/helper.js";
+
+const addBook = async (reqBody) => {
+  const {
+    title,
+    authors,
+    ISBN,
+    category,
+    publicationYear,
+    totalCopies,
+    shelfNumber,
+  } = reqBody;
+  const authorsArr = authors.split(",");
+
+  try {
+    const BookExistCheck = await helper.bookExistingCheck(ISBN);
+
+    if (BookExistCheck) {
+      const error = new Error("BOOK_EXIST");
+      throw error;
+    }
+
+    const data = await bookMdl.book({
```

Comment:
aa Model nu pan bov var kidhu che.
BookMdl avse instead of bookMdl.
and BookMdl.find , BookMdl.create - avu avse.
`bookMdl.book.find` avu nai kar.

aavu badha model mate same vastu kari deje fix.

Reply:
okay mne lgyu camel case rkhvu pdse atle